### DISCUSSION TITLE
Regression Test data assembly

### DISCUSSION THREAD

**Discussion by vondele** [official-stockfish/Stockfish#3700] [Q&A] [answered]
Leonardo Ljubičić has been maintaining our regression test webpages since a long time (https://github.com/glinscott/fishtest/wiki/Regression-Tests). This is really valuable information for the project, many thanks!

Now, we're looking for a successor in this role....  
Who could maintain the corresponding excel sheet, and copy and paste the graphs to the wiki pages?
This is a great way for long term fans to contribute :-) 

The current excel sheet can be downloaded from https://docs.google.com/spreadsheets/d/1aogt_cd8C_T-xFuLHW5qYCwfB03ha0IG/edit?usp=sharing&ouid=117220288956416990756&rtpof=true&sd=true

**Comment by xoto10** [comment]
I guess Leonardo uses Windows, do the spreadsheet graphs look correct for other people in other OS?
It doesn't display well on my Chromebook, and I'm struggling to download the raw xlsx to look at it in LibreOffice on my desktop. Is there a trick to downloading from google docs that I'm missing, or can we put it on a different file server?

**Reply by vondele** [reply]
I believe that it needs microsoft excel (so windows, mac) to show the graphs properly, at least didn't work here on libreoffice or as a google doc either. Or somebody could host it at office 365 or similar? 
(The following link is the same document uploaded as a .zip https://drive.google.com/file/d/1i0ULgk4ByujS9jQk47KcVvVdNDCcE8oI/view?usp=sharing in case that changes the download process)

**Reply by xoto10** [reply]
Ah, that helped. I downloaded the zip and that seemed to work ok, I viewed the graphs more or less as displayed in the Regression page. :-)
That was using LibreOffice (version ???) on Ubuntu 18. Interesting that LibreOffice didn't work for you.

**Comment by bknox83** [comment]
I might be interested if no one else is jumping to do it. I'll need a few days to digest the charts and excel setup so that they make sense to me. I'll post a link to a wiki fork when I feel it's ready for review. There are a few QoL updates I'd like to propose.

**Reply by bknox83** [reply]
@vondele Looks like it wasn't nearly as complicated as I was thinking it might be so I finished up my rough drafts pretty quickly.

I have an updated excel file that I modified into a format that I understood. Basically just a little bit of reformatting from Leonardo's document. I can share it wherever it makes the most sense. For now it's living on my hard drive and google drive.

Also, the fork of my regression test page can be found here with small formatting updates - https://github.com/bknox83/fishtest/wiki/Regression-Tests

Let me know if these are acceptable or if you want to change anything.

**Reply by vondele** [reply]
great!

Minor nits:
* I would refer to current master as SF15dev or maybe SF14+ and reserve 'SF15' for the released version
* I like to see the data points as 'symbols' on the curves, it gives a bit a better impression on how regularly things are test. But no strong opinions there .... those that do decide.

**Reply by bknox83** [reply]
Thanks! Good ideas. I know there was talk of maybe doing 14.x releases at some point so I changed all references of SF15 to SF14+. Also added symbols to the curves for when tests occurred.

**Reply by vondele** [reply]
this looks very nice to me... ready to go IMO.

**Reply by bknox83** [reply]
Sounds good! I'll plan on updating the wiki after the next progression test. You can reach me here or I'm also in the Discord (Brad).

**Reply by snicolet** [reply]
Thanks Brad! :-)

**Comment by vondele** [comment, marked as answer]
thanks at @bknox83 !
